source/_integrations/sms.markdown

```diff
@@ -32,6 +32,7 @@ You can also enable `SMS` via your `configuration.yaml` file:
 # Example configuration.yaml entry
 sms:
   device: /dev/ttyUSB2
+  baud_speed: 19200 # Optional
```

Comment:
For integrations that connect to devices or services, we no longer accept new YAML configuration or changes.

More information about this can be found in Architecture Decision Record:

- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>